src/GitHub.TeamFoundation.14/Connect/GitHubConnectSection.cs

```diff
@@ -216,8 +216,13 @@ void OnPropertyChange(object sender, PropertyChangedEventArgs e)
                         HandleClonedRepo(newrepo);
 
                     isCreating = isCloning = false;
-                    var repo = await ApiFactory.Create(newrepo.CloneUrl).GetRepository();
-                    newrepo.SetIcon(repo.Private, repo.Fork);
+
+                    try
+                    {
+                        var repo = await ApiFactory.Create(newrepo.CloneUrl).GetRepository();
+                        newrepo.SetIcon(repo.Private, repo.Fork);
+                    }
+                    catch { }
```

Comment:
Let's add a TODO here to add logging later on, and a comment explaining that `GetRepository()` may throw if the user doesn't have permissions to access the repo (because the repo no longer exists, or because the user has logged in on a different profile, or their permissions have changed remotely)